bridge/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 // For more info on versioning, see the README.
-val version = "0.1.0"
+val version = "0.2.0"
```

Comment:
Does there need to be an entry added to `CHANGELOG` for this version change? Is artifact generation happening automatically or do we need to run assemble manually for each PR with a version change?

Reply:
Great callout here.

1. Yep, a CHANGELOG update is required here. Thanks for the callout. This builds good SDK release habits! Added: a14b78c08dd8e3683122230354ab9e9011df77a9
2. Currently its all manual. My plan is to have a separate PR that includes the updated `.aar` in`/app` along with corresponding `/app` changes.

Reply:
Do we need to update the version actually right now? Could we keep just using "snapshot" builds until we're ready to "release 0.1.0? I don't think we should bump the version until we know where we are releasing them.

Reply:
Brian and I talked about this offline, and we decided to leave the version at 0.1.0 for now and keep calling every build a `SNAPSHOT` build. 

Then once we are in a stable place, we can bump the version to 1.0.0 and update the CHANGELOG accordingly.

Reply:
58d6f3e4716d47460e0aad390f1dc0a2d5a2bde0